cypress/integration/assign-automatic-tag.spec.js

```diff
@@ -0,0 +1,45 @@
+import * as data from '../../cypress.json';
+describe('Assign Automatic tags', function() {
+    beforeEach(function() {
+        cy.login(data.email, data.password);
+        // cy.visit(data.baseUrl + "/core/setting/tagsetting");
+    });
+    afterEach(function() {
+        cy.logout();
+    });
+    //it should visit setting/imap seting page
+    it('Add imap email and change their status', function () {
+        cy.addImap();
+        cy.get("#switchState").click();
+    })
+    // add a job probile
+    it('Add a job profile', function () {
+        cy.addJobprofile();
+    })
+
+    //visit inbox page and fetch latest emails
+    it('visit inbox page and fetch latest email', function () {
+        cy.server()
+        cy.route('GET',`http://localhost:8091/email/fetchByButton**`).as('fetch_emails')
```

Comment:
why is url hard coded here?

Reply:
ok sir i will remove the hard coded part